tests/acceptance/features/apiProvisioningLDAP/userSync.feature

```diff
@@ -22,3 +22,29 @@ Feature: Single user sync using the OCS API
       | ocs-api-version | ocs-status-code |
       | 1               | 100             |
       | 2               | 200             |
+
+  Scenario Outline: admin syncs after changing display name of a ldap user
+    Given using OCS API version "<ocs-api-version>"
+    When the administrator sets the ldap attribute "displayname" of the entry "uid=user0,ou=TestUsers" to "ldap user"
```

Comment:
We could also change `displayname` of `user1` to some different string like `name of user one`
And then after sync of `user0` check that the display name of `user1` is still its original value (probably `User One` or `user`)
That will demonstrate that only the attributes of `user0` were synced.

Reply:
OK that's very nice